test/images/resource-consumer/Dockerfile

```diff
@@ -12,7 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jess/stress
+FROM BASEIMAGE
```

Comment:
@luxas the pattern I've seen in the repo seems to be using `BASEIMAGE` in the Dockerfile, regardless of whether there's a simple pattern for the base image (as is the case with our custom base images). I'm fine with continuing this pattern.